### PR TITLE
Adding support for expanded property for Fitbit sleeps

### PR DESCRIPTION
**Pull request by Uthkarsh**

### CODE REVIEW

**Review by aviade — APPROVED** [review]

File: ValidicCSharp/Model/FitBitModel/FitBitExpandedSleep.cs [hunk 1]
```diff
@@ -0,0 +1,23 @@
+﻿namespace ValidicCSharp.Model
+{
+    using System;
+    using Newtonsoft.Json;
+
+    public class FitBitExpandedSleep : Sleep
```
Comment (jkrez) [inline ValidicCSharp/Model/FitBitModel/FitBitExpandedSleep.cs]: "Fitbit" is one word. Please change the capitalization and lowercase the "B"

File: ValidicCSharp/Model/FitBitModel/FitBitExpandedSleep.cs [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        public bool? is_main_sleep { get; set; }
+
+        [JsonProperty("start_time")]
+       public DateTimeOffset? Sleep_Start_time { get; set; }
+
+        [JsonProperty("minutes_to_fall_asleep")]
+        public Double? minutes_to_fall_asleep { get; set; }
+
+        [JsonProperty("minutes_after_wakeup")]
+        public Double? minutes_after_wakeup { get; set; }
```
Comment (jkrez) [inline ValidicCSharp/Model/FitBitModel/FitBitExpandedSleep.cs]: Please change all the properties to use the same naming scheme as other files in the project: using pascal case and no underscores. Follow the examples in ValidicCSharp/Model/Sleep.cs for exmaple

File: ValidicCSharp/ValidicCSharp.csproj [hunk 3]
```diff
@@ -50,6 +50,7 @@
     <Compile Include="Model\AddUserRequest.cs" />
     <Compile Include="Model\Application.cs" />
     <Compile Include="Model\Credentials.cs" />
+    <Compile Include="Model\FitBitModel\FitBitExpandedSleep.cs" />
```
Comment (jkrez) [inline ValidicCSharp/ValidicCSharp.csproj]: Please rename "FitBitModel" to "FitbitModel" as "Fitbit" is a single word

**Review by jkrez — CHANGES_REQUESTED** [review]

File: ValidicCSharp/Model/FitBitModel/FitBitExpandedSleep.cs (hunk 1 above)
Comment (Uthkarsh) [reply ValidicCSharp/Model/FitBitModel/FitBitExpandedSleep.cs]: Yes, I shall make those changes.

Comment (jkrez) [reply ValidicCSharp/Model/FitBitModel/FitBitExpandedSleep.cs]: Thanks!

**Review by jkrez — COMMENTED** [review]
Thanks for doing this!

**Comment by jkrez** [comment]
I update the directory name. Everything looks good now!

**Pull request merged**